Bottom: 57ff8c924d86c8267d0a9dada7dc4a5bc42398a8
Top:    44e8ba48cb74ca3d3dccedb62d936abe1b9561d0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 02:27:06 +0100

Refresh of hid-gtk-remove-unused-extern-h

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 3d3896f..9fd146e 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -218,8 +218,6 @@ static ToggleFlagType *tflags = 0;
 static int n_tflags = 0;
 static int max_tflags = 0;
 
-extern HID ghid_hid;
-
 GhidGui _ghidgui, *ghidgui = NULL;
 
 GHidPort ghid_port, *gport;
